### ISSUE TITLE
Using details/summary in place of accordions and disclosure elements

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#814]
A meta issue to discuss when the native HTML details / summary element could or should be used in LocalGov Drupal . 

This came up in the Accessibility Governance Group, and have requested that we collate any recent and up to date resources and examples . Then we can decide what to do next. 

Related to @millnut 's work on https://github.com/localgovdrupal/localgov_base/pull/708 and an unrelated discussion on slack .

**Comment by waako** [comment]
Some relevant resources for reference:

- [Progressively Enhanced HTML Accordion by Adrian Roselli](https://adrianroselli.com/2023/08/progressively-enhanced-html-accordion.html)
- [The details and summary elements, again by Scott O'Hara](https://www.scottohara.me/blog/2022/09/12/details-summary.html)
- [details/summary inconsistencies by Matuzo](https://www.matuzo.at/blog/2023/details-summary/)

**Comment by msayoung** [comment]
Pulling some resources from a comment thread on Slack ( https://localgovdrupal.slack.com/archives/C045Z40KGH0/p1746176186926499 ) 

NikLP (c/o PopPop) - May 2nd at 10:33 AM
Oh h2 inside summary is a no no https://blog.logrocket.com/styling-html-modern-css/

Jonny Holden (Calderdale / poppop) - May 6th at 4:47 PM
@Maria (Agile Collective)
everything I know about them is here... https://adrianroselli.com/2020/05/disclosure-widgets.html (always excellent, easy to read, and kept up to date) https://www.scottohara.me/blog/2022/09/12/details-summary.html

**Comment by millnut** [comment]
Headings are allowed within summary, see https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/summary#summaries_as_headings

**Comment by msayoung** [comment]
From gov.uk

- https://design-system.service.gov.uk/components/accordion/
- https://design-system.service.gov.uk/components/details/

(thanks @FrazerGNom )

**Comment by tonypaulbarker** [comment]
Thanks all. 

Moved to Drupal gitlab

https://git.drupalcode.org/project/localgov_base/-/work_items/3590894
